http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -647,12 +629,7 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
 
       // Save or retrieve a KubernetesCluster record for the app
       saveCluster <- F.fromEither(
-        getSavableCluster(userInfo.userEmail,
-                          cloudContext,
-                          ctx.now,
-                          landingZoneResourcesOpt.map(_.clusterName),
-                          landingZoneResourcesOpt.map(_.region)
-        )
+        getSavableCluster(userInfo.userEmail, cloudContext, ctx.now)
       )
       saveClusterResult <- KubernetesServiceDbQueries.saveOrGetClusterForApp(saveCluster).transaction
```

Comment:
There's a DB optimization I want to make in here too which I haven't pushed yet.. will update this comment to explain.